README.md

```diff
@@ -1 +1,91 @@
-# math-magicians
\ No newline at end of file
+### Math Magician
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## About the Project
+
+> This project is built with React, it is practicing application to build a calculator using React.js.
+
+## Built With
+
+- React.js
+- Jest
+- Gitflow
+
+### Key Features
+
+- Page Setup
+
+<!-- ## Live Demo
+
+not available yet -->
+
+## Getting Started
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-prefer-folder
+  git clone https://github.com/saluumaa/Math-magicians
+```
+
+### Run this command to run server
+
+> npm start
+
+### Run this command build server
+
+> npm run build
+
+### Run this command to run tests
+
+> npm run test
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Salma Ibrahim**
+
+- GitHub: [@Saluumaa](https://github.com/saluumaa)
+- Twitter: [@SalmaHIbrahim20](https://twitter.com/SalmaHIbrahim20)
+- LinkedIn: [Salma ibrahim](https://www.linkedin.com/in/salma-ibrahim-78bb5a14a/)
+
+## 🔭 Future Features
+
+- 3 sub-pages
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/saluumaa/Math-magicians/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 🙏 Acknowledgments
+
+I would like to thank Microverse!
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- Great job @saluumaa :muscle: Your setup is complete. However, the MIT license link in your README leads to an error 404 page. I think this is because you don't have a License file in your project. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink: 

![Screenshot from 2023-03-27 11-34-45](https://user-images.githubusercontent.com/102699958/227931952-cd1975ed-14d8-4763-aa16-f1fc69c07a27.png)